src/apps/components/AppAlerts/utils.test.ts

```diff
@@ -0,0 +1,143 @@
+import { EventDeliveryStatusEnum } from "@dashboard/graphql";
+
+import { getLatestFailedAttemptFromWebhooks } from "./utils";
+
+describe("getLatestFailedAttemptFromWebhook", () => {
+  it("should return the latest failed attempt from failedDelivers", () => {
+    // Arrange
+    const webhook: any = {
```

Comment:
suggestion: avoid using `any` when you have types generated for this, using `as Somthing` is better